badge_server/test_badges.py

```diff
@@ -0,0 +1,206 @@
+from compatibility_lib import package
+
+import main
+import utils
+
+APACHE_BEAM_RECENT_SUCCESS_2 = compatibility_store.CompatibilityResult(
+    [package.Package('apache-beam[gcp]')],
+    python_major_version=2,
+    status=compatibility_store.Status.SUCCESS,
+    timestamp=datetime.datetime.utcnow())
+
+GOOGLE_API_CORE_RECENT_SUCCESS_2 = compatibility_store.CompatibilityResult(
+    [package.Package('google-api-core')],
+    python_major_version=2,
+    status=compatibility_store.Status.SUCCESS,
+    timestamp=datetime.datetime.utcnow())
+
+GOOGLE_API_CORE_RECENT_SUCCESS_3 = compatibility_store.CompatibilityResult(
+    [package.Package('google-api-core')],
+    python_major_version=3,
+    status=compatibility_store.Status.SUCCESS,
+    timestamp=datetime.datetime.utcnow())
+
+TENSORFLOW_RECENT_SUCCESS_3 = compatibility_store.CompatibilityResult(
+    [package.Package('tensorflow')],
+    python_major_version=3,
+    status=compatibility_store.Status.SUCCESS,
+    timestamp=datetime.datetime.utcnow())
+
+APACHE_BEAM_GOOGLE_API_CORE_RECENT_SUCCESS_2 = compatibility_store.CompatibilityResult(
+    [package.Package('google-api-core'),
+     package.Package('apache-beam[gcp]')],
+    python_major_version=2,
+    status=compatibility_store.Status.SUCCESS,
+    timestamp=datetime.datetime.utcnow())
+
+APACHE_BEAM_GOOGLE_API_PYTHON_CLIENT_RECENT_SUCCESS_2 = compatibility_store.CompatibilityResult(
+    [
+        package.Package('google-api-python-client'),
+        package.Package('apache-beam[gcp]')
+    ],
+    python_major_version=2,
+    status=compatibility_store.Status.SUCCESS,
+    timestamp=datetime.datetime.utcnow())
+
+GOOGLE_API_CORE_GOOGLE_API_PYTHON_CLIENT_RECENT_SUCCESS_2 = compatibility_store.CompatibilityResult(
+    [
+        package.Package('google-api-core'),
+        package.Package('google-api-python-client')
+    ],
+    python_major_version=2,
+    status=compatibility_store.Status.SUCCESS,
+    timestamp=datetime.datetime.utcnow())
+
+GOOGLE_API_CORE_GOOGLE_API_PYTHON_CLIENT_RECENT_SUCCESS_3 = compatibility_store.CompatibilityResult(
+    [
+        package.Package('google-api-core'),
+        package.Package('google-api-python-client')
+    ],
+    python_major_version=3,
+    status=compatibility_store.Status.SUCCESS,
+    timestamp=datetime.datetime.utcnow())
+
+GOOGLE_API_CORE_TENSORFLOW_RECENT_SUCCESS_3 = compatibility_store.CompatibilityResult(
+    [package.Package('google-api-core'),
+     package.Package('tensorflow')],
+    python_major_version=3,
+    status=compatibility_store.Status.SUCCESS,
+    timestamp=datetime.datetime.utcnow())
+
+GOOGLE_API_PYTHON_CLIENT_TENSORFLOW_RECENT_SUCCESS_3 = compatibility_store.CompatibilityResult(
+    [
+        package.Package('google-api-python-client'),
+        package.Package('tensorflow')
+    ],
+    python_major_version=3,
+    status=compatibility_store.Status.SUCCESS,
+    timestamp=datetime.datetime.utcnow())
+
+
+class TestBadgeServer(unittest.TestCase):
```

Comment:
Seems like these tests should belong in test_badge_server.py?

Reply:
I think that these tests will end up replacing those and I was imagining splitting the tests into two modules: one for the image output and one for the HTML output.

Reply:
ok sgtm